tests/Context/ContextTest.php

```
@@ -1140,4 +1140,1277 @@ public function testRefreshClearsAssignmentCacheForExperimentIdChange(): void {
+		$context->publish();
+
+		self::assertCount(1, $this->eventHandler->submitted);
+		$event = $this->eventHandler->submitted[0];
+		self::assertSame(count($experiments), count($event->exposures));
+	}
+
+	public function testAttributeUpdatesInTemplates(): void {
+		$context = $this->createReadyContext('audience_context.json');
+
+		$context->setAttribute('age', 15);
+		$treatmentBefore = $context->getTreatment('exp_test_ab');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$treatmentBefore'. (undefined)

(UnusedLocalVariable)